Template/README.md

```diff
@@ -1 +1,18 @@
-# {PROJECT}
\ No newline at end of file
+# {PROJECT}
+
+### Installation
+
+**CocoaPods:**
+Add the line `pod "{PRODUCT}"` to your `Podfile`
+
+**Carthage:**
+
+Add the line `github "{URL}/{PRODUCT}"` to your `Cartfile`
+
+**Manual:**
+
+Clone the repo and drag the file `{PRODUCT}.swift` into your Xcode project.
+
+**Swift Package Manager:**
+
+Add the line `.Package(url: "{URL}.git", majorVersion: 2)` to your `Package.swift` file.
```

Comment:
Sorry, I think I gave you incorrect feedback earlier, we **shouldn't** set the major version to `2` by default - instead let's use `1`.